composition-js/src/validate.ts

```diff
@@ -64,25 +69,40 @@ export class ValidationError extends Error {
   }
 }
 
-function validationError(
+
+function satisfiabilityError(
   unsatisfiablePath: RootPath<Transition>,
   subgraphsPaths: RootPath<Transition>[],
   subgraphsPathsUnadvanceables: Unadvanceables[]
-): ValidationError {
+): GraphQLError {
   const witness = buildWitnessOperation(unsatisfiablePath);
-
```

Comment:
Remark: removed that TODO because it is outdated, we do have more detailed error messages now. Technically unrelated to that PR though.